### ISSUE TITLE
Solr deprecation warnings

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_blacklight#101]
In GitLab by @cdm32 on Sep 14, 2018, 07:38

Just noticed these this morning, don't believe they've cropped up before

![2018-09-14_0737](https://user-images.githubusercontent.com/6477867/46040357-0a20e200-c0de-11e8-9092-7bae3a8ce91d.png)

```
9/13/2018, 12:30:24 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.TrieIntField]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:24 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.TrieFloatField]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:24 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.TrieLongField]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:24 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.TrieDoubleField]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:24 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.TrieDateField]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:25 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.SynonymFilterFactory]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:25 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.WordDelimiterFilterFactory]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:25 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.LatLonType]. Please consult documentation how to replace it accordingly.
9/13/2018, 12:30:25 PM
WARN false
x:blacklight-core
SolrResourceLoader
Solr loaded a deprecated plugin/analysis class [solr.GeoHashField]. Please consult documentation how to replace it accordingly.
```

**Comment by banukutlu** [comment]
In GitLab by @cdm32 on Sep 14, 2018, 08:26

changed the description

**Comment by banukutlu** [comment]
http://lucene.apache.org/solr/7_4_0/solr-core/deprecated-list.html

**Comment by banukutlu** [comment]
Re Deprecated Trie* field from https://github.com/apache/lucene-solr/blob/d7fd82c0f8517251d67b0af021d259dffaa4dce6/solr/CHANGES.txt#L760

> * All Trie* numeric and date field types have been deprecated in favor of *Point field types.
  Point field types are better at range queries (speed, memory, disk), however simple field:value queries underperform
  relative to Trie. Either accept this, or continue to use Trie fields.
  This shortcoming may be addressed in a future release.

**Comment by banukutlu** [comment]
moving to MVP1 to be revisited after the BL 7.0 release.

**Comment by banukutlu** [comment]
BL 7.0 will fix Trie* field warnings 🎉 `https://github.com/projectblacklight/blacklight/pull/2014`

**Comment by cdmo** [comment]
Open Source 💫

**Comment by banukutlu** [comment]
Fixed by #175
